src/libkrun/src/lib.rs

```diff
@@ -752,8 +731,172 @@ pub unsafe extern "C" fn krun_set_data_disk(ctx_id: u32, c_disk_path: *const c_c
     KRUN_SUCCESS
 }
 
+/*
+ * Send the VFKIT magic after establishing the connection,
+ * as required by gvproxy in vfkit mode.
+ */
+#[cfg(feature = "net")]
+const NET_FLAG_VFKIT: u32 = 1 << 0;
+
+/* Taken from uapi/linux/virtio_net.h */
+#[cfg(feature = "net")]
+const NET_VFLAG_CSUM: u32 = 1 << 0;
+#[cfg(feature = "net")]
+const NET_VFLAG_GUEST_CSUM: u32 = 1 << 1;
+#[cfg(feature = "net")]
+const NET_VFLAG_GUEST_TSO4: u32 = 1 << 7;
+#[cfg(feature = "net")]
+const NET_VFLAG_GUEST_TSO6: u32 = 1 << 8;
+#[cfg(feature = "net")]
+const NET_VFLAG_GUEST_UFO: u32 = 1 << 10;
+#[cfg(feature = "net")]
+const NET_VFLAG_HOST_TSO4: u32 = 1 << 11;
+#[cfg(feature = "net")]
+const NET_VFLAG_HOST_TSO6: u32 = 1 << 12;
+#[cfg(feature = "net")]
+const NET_VFLAG_HOST_UFO: u32 = 1 << 14;
+#[cfg(feature = "net")]
+const NET_COMPAT_VFLAGS: u32 = NET_VFLAG_CSUM
+    | NET_VFLAG_GUEST_CSUM
+    | NET_VFLAG_GUEST_TSO4
+    | NET_VFLAG_GUEST_UFO
+    | NET_VFLAG_HOST_TSO4
+    | NET_VFLAG_HOST_UFO;
+#[cfg(feature = "net")]
+const NET_ALL_VFLAGS: u32 = NET_VFLAG_CSUM
+    | NET_VFLAG_GUEST_CSUM
+    | NET_VFLAG_GUEST_TSO4
+    | NET_VFLAG_GUEST_TSO6
+    | NET_VFLAG_GUEST_UFO
+    | NET_VFLAG_HOST_TSO4
+    | NET_VFLAG_HOST_TSO6
+    | NET_VFLAG_HOST_UFO;
```

Comment:
It is not clear what are the compat flags - maybe add a comment? Seems that the difference are NET_VFLAG_GUEST_TSO6 and NET_VFLAG_HOST_TSO6.

We can use this to make this more clear:

```rust
const NET_ALL_VFLAGS: u32 = NET_COMPAT_VFLAGS
    | NET_VFLAG_GUEST_TSO6
    | NET_VFLAG_HOST_TSO6;
```

Reply:
Added a comment clarifying why that's there, thanks!